index.js

```diff
@@ -407,6 +407,26 @@
         }
     }
 
+    // ----------------------------------------
+    // TimeChecker
+    // ----------------------------------------
+    class TimeChecker {
+        static checkTime = async () => {
+            console.log("hallo")
+            const ntpResponse = await fetch("https://worldtimeapi.org/api/timezone/est")
+            const data = await ntpResponse.json()
+            const ntpDate = data['unixtime'] * 1000
+            console.log("test1")
```

Comment:
this seems like it should be removed

Reply:
You are absolutely right. Removed my bad debugging